### ISSUE TITLE
Extend the narrative parts for the profiles and extensions of the IG

### ISSUE THREAD

**Issue by regelkraut** [HL7Austria/HL7-AT-FHIR-Core-R4#16]
**Real Submitter**
FH Johanneum, Carina Seerainer

**Resource(s)/Profiles incl. Version**
HL7 AT FHIR Core IG STU1

**Links**
https://hl7austria.github.io/r4-core-ballot_2021_01/artifacts.html

**Priority**
Major

**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
More descriptive text would be helpful, such as introductory words, aim of the guide, etc. The resources/extensions, for example AustrianAddress, do not make it clear what the extension (additional content) should/can be used for.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by regelkraut** [comment]
**Ballot Info:** Ballot 2021-1, comment nr. 148

**Ballot Weight:** Z-K (approval with comment)

**Proposed Solution from the "Ballot-Leitkreis:"**
The narrative of the profiles and extensions will be reworked, also a governance for creating narratives for IGs will be creating in the TC FHIR.

**Comment by oliver-krauss** [comment]
Suggestion: 
- Governance is not part of the ballot - we will create a new issue with guidance that will be discussed in the next TCs
- Narrative will be updated for the main page (why do we need core) + for every profile/extension a short description.
  - The narrative will be verified during the "wiederspruchs" phase by all balloters who can again comment.
(note: comment will be updated later with the correct english wording)

Discussed on Nov 18

**Comment by oliver-krauss** [comment]
@aschuler84 -> Tomorrows call. Please check this page:
https://fhir.hl7.at/r4-core-main/artifacts.html
Should there be more info on this page? If so - it is auto-generated we would need to override in our base-template

**Comment by oliver-krauss** [comment]
This can only be closed after #42 and #41 are ALSO closed
